frontend/components/News/ArticleRequest.tsx

```diff
@@ -0,0 +1,208 @@
+import EditOutlinedIcon from '@mui/icons-material/EditOutlined';
+import
+{
+  Avatar, Box,
+  Button,
+  IconButton, Paper, Stack, Typography,
+} from '@mui/material';
+import Grid from '@mui/material/Grid';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'next-i18next';
+import Image from 'next/image';
+import { useRef } from 'react';
+import Markdown from '~/components/Markdown';
+import
+{
+  authorIsUser,
+  getAuthor,
+  getAuthorStudentId,
+  getSignature,
+} from '~/functions/authorFunctions';
+
+import { timeAgo } from '~/functions/datetimeFunctions';
+import selectTranslation from '~/functions/selectTranslation';
+import
+{
+  ArticleRequest as ArticleRequestType,
+  useApproveRequestMutation, useRejectRequestMutation,
+  Author,
+} from '~/generated/graphql';
+import { hasAccess, useApiAccess } from '~/providers/ApiAccessProvider';
+import { useUser } from '~/providers/UserProvider';
+import routes from '~/routes';
+import Link from '../Link';
+import Tag from '../Tag';
+import articleStyles from './articleStyles';
+
+type ArticleProps = {
+  article: Partial<Omit<ArticleRequestType, 'author'> & { author: Partial<Author> }>;
+  refetch?: () => void;
+  rejected?: boolean;
+};
+
+export default function ArticleRequest({
```

Comment:
It would be nice to be able to undo an article rejection. I.e if you accidentally reject an article.

Reply:
Should refetch when an article is rejected, otherwise the rejected list is empty after rejecting an article (until page refresh).